backend/routes/agency.js

```diff
@@ -0,0 +1,57 @@
+const express = require('express');
+const mongoose = require('mongoose');
+const { Agency } = require('../models');
+const { body, validationResult } = require('express-validator');
+const router = express.Router();
+
+const validationChain = [
+    body('tableContent.phone').trim().isMobilePhone('en-US'),
+    body('primaryContact').trim().isMobilePhone('en-US'),
+    body('billingZipcode').trim().isPostalCode('US'),
+    body('contacts.*.phoneNumber').trim().isMobilePhone('en-US'),
+    body('contacts.*.email').trim().isEmail(),
+    body('scheduledNextVisit').isISO8601(),
+    body('dateOfMostRecentAgreement').trim().isDate({ format: 'MM/DD/YYYY' }),
+    body('dateOfInitialPartnership').trim().isDate({ format: 'MM/DD/YYYY' }),
+    body('fileAudit').trim().isDate({ format: 'MM/DD/YYYY' }),
+    body('monitored').trim().isDate({ format: 'MM/DD/YYYY' }),
+    body('foodSafetyCertification').trim().isDate({ format: 'MM/DD/YYYY' }),
+];
+
+router.put('/', validationChain, async (req, res, next) => {
```

Comment:
Can comments be added to these routes to describe what they're for

Reply:
Seconding this comment